### ISSUE TITLE
test KM rendering

### ISSUE THREAD

**Issue by acthp** [ucscXena/ucsc-xena-client#158]
Need a unit test for rendering KmPlot component, with different data types (phenotype, matrix, mutation, segmented). Use the react hooks to test rendering, as in the plotDenseMatrix test.

**Comment by souravsingh** [comment]
@acthp I am interested in working on the issue. We have to take help from the plotDenseMatrix test and create a test for KmPlot, correct?

**Comment by acthp** [comment]
@souravsingh Correct. Initially, the main thing is just to see that it renders at all. An first step is to review the parameters required by the component, which can be done by putting a console.log(this.props) in the KmPlot render method, and asking questions if something isn't clear.

**Comment by souravsingh** [comment]
@acthp I have checked the parameters for the KmPlot render method and it seems to require xScale, yScale, color, label, curve, setActiveLabel and isActive. Now do we set the values for these parameters?

My apologies for the question, I am a beginner at JavaScript.

**Comment by acthp** [comment]
@souravsingh I'm not sure if you mean "how do I pass parameters to KmPlot?", or "what are the valid values for these parameters?" Also, xScale, yScale, etc., are the props for LineGroup, which is a child component of KmPlot. We want, instead, to test KmPlot. KmPlot (toward the bottom of the file) takes props km, dims, and callback. 

To pass the parameters to KmPlot, you would do something like

```
var km = {...};
renderIntoDocument(<KmPlot km={km} dims={...} ... />);
```

The ```km``` properties will take some investigation to understand. The ```callback``` property can probably be skipped, because it's only for interactive use.

**Comment by souravsingh** [comment]
My apologies for the improperly phrased question, I wanted to know about the valid parameters that we needed to pass for the tests.
